<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSpotlightSetting</class>
 <widget class="QDialog" name="QSpotlightSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Spotlight Shortcut Key:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Readmode Shortcut  Key:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lasterpen Shortcut Key:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>FindMouse Shortcut Key:</string>
   </property>
  </widget>
  <widget class="QSlider" name="sliderWatordrop">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="sliderSpotLight">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>10</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>102</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lasterpen Radius:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>102</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Spotlight Radius:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>102</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>FindMouse Radius:</string>
   </property>
  </widget>
  <widget class="QSlider" name="sliderLasterPen">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>35</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>120</y>
     <width>96</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Lasterpen Color:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>91</y>
     <width>96</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Spotlight Color:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="colorLasterpen">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>120</y>
     <width>32</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>149</y>
     <width>96</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>FindMouse Color:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="colorWatordrop">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>149</y>
     <width>32</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="colorSpotlight">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>91</y>
     <width>32</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>102</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>102</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>Language Type:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="LanguagecomboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>0</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>English</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chinese</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>360</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_active_value">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>102</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Active value:</string>
   </property>
  </widget>
  <widget class="QSlider" name="sliderActiveValue">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="RestoreDefaultpushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>410</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Restore default settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Screenshot Shortcut Key:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Magnifying Shortcut Key:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>140</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>102</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Readmode Setting:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonLeft">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonRight">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>320</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonNormal">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
